### DISCUSSION TITLE
Let's invite Bart Bonte and Rusty Lake to showcase their games on airsdk.dev

### DISCUSSION THREAD

**Discussion by GeneralVimes** [airsdk/Adobe-Runtime-Support#2382] [Ideas]
There are prominent AIR game developers who I know of:
Bart Bonte: https://www.bontegames.com/
Rusty Lake: https://www.rustylake.com/

Their games were among the Google Play's best games and also featured on Apple. Maybe @ajwfrost could invite them to showcase their games on airsdk.dev? Just to have this invitation look more "official". Such games examples can certainly do well to attract new developer to the technology.

**Comment by marchbold** [comment]
I'll reach out to them and see if they are okay with us including any of their games on the site.

**Comment by KupoGames** [comment]
I'm a prominent AIR developer too!
My games Epic Battle Fantasy 4 and 5 are some of the highest-rated indie games on Steam: https://store.steampowered.com/app/432350/Epic_Battle_Fantasy_5/
Some of my games are made in Starling, but most are just classic display list. Most are AS3, but some are even made with AS2!
I'm still very new to mobile development, but EBF5 is out on Android and iOS too.

Feel free to include any of my games. :P

**Reply by marchbold** [reply]
Great news! 

Could you supply me with an image, title and a description for the showcase item? (Some examples here: https://airsdk.dev/showcase)

**Reply by KupoGames** [reply]
Here you go: https://www.dropbox.com/s/l97y81l34e5h7f9/AIR%20showcase%20Epic%20Battle%20Fantasy%20games.zip?dl=0
I included the info for 4 games, all of which show different things that AIR/Flash can do.

**Reply by marchbold** [reply]
Brilliant! I'll get these up on the site. (Apologies for the delayed response, been a busy few weeks getting all the extensions updated).
